spec/bundler/shared_helpers_spec.rb

```diff
@@ -170,6 +170,105 @@
       end
     end
   end
+  describe "#in_bundle?" do
+    it "calls to find_gemfile" do
```

Comment:
this is an implementation detail -- please actually test the return value of `in_bundle?`